products/compute/terraform.yaml

```diff
@@ -627,26 +627,29 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           health_check_name: "internal-service-health-check"
     custom_code: !ruby/object:Provider::Terraform::CustomCode
       encoder: templates/terraform/encoders/health_check_type.erb
+      decoder: templates/terraform/decoders/health_check.erb
+      constants: templates/terraform/constants/health_check.erb
+      resource_definition: templates/terraform/resource_definition/health_check.erb
     properties:
      id: !ruby/object:Overrides::Terraform::PropertyOverride
        exclude: true
      type: !ruby/object:Overrides::Terraform::PropertyOverride
        output: true
        description: The type of the health check. One of HTTP, HTTPS, TCP, or SSL.
-     httpHealthCheck.portName: !ruby/object:Overrides::Terraform::PropertyOverride
-       exclude: true
+     # httpHealthCheck.portName: !ruby/object:Overrides::Terraform::PropertyOverride
```

Comment:
Is there a reason we have these commented out?

Reply:
Oh, yes, sorry.  I meant to ask why portName is excluded.  `port_name` will need to be available for `USE_NAMED_PORT`.  Perhaps there is another way to make it available?